x86/boards.txt

```diff
@@ -1,7 +1,87 @@
-virtual.name="Kaleidoscope Virtual Keyboard"
-virtual.build.usb_product="Kaleidoscope Virtual Keyboard"
-virtual.build.usb_manufacturer="Kaleidoscope"
-virtual.build.board=VIRTUAL
-virtual.build.core=virtual
-virtual.build.variant=virtual
-virtual.build.extra_flags=-DKALEIDOSCOPE_HARDWARE_H="Kaleidoscope-Hardware-Virtual.h"
+# See: http://code.google.com/p/arduino/wiki/Platforms
+
+#menu.cpu=Processor
+#menu.bootloader=Bootloader
+#menu.pinout=Pinout
+
+##############################################################
+
+model01.name=Keyboardio Model 01
+model01.build.usb_product="Model 01"
+model01.build.usb_manufacturer="Keyboardio"
+model01.build.board=AVR_MODEL01
+model01.build.core=virtual
+model01.build.variant=model01
+model01.build.extra_flags={build.usb_flags} '-DKALEIDOSCOPE_HARDWARE_SPEC_H="kaleidoscope/hardware/keyboardio/Model01_Spec.h"' '-DKALEIDOSCOPE_MCU_H="mcu/atmega32u4.h"'
+
+keyboardio_imago.name=Keyboardio Imago
+keyboardio_imago.build.usb_product="Imago"
+keyboardio_imago.build.usb_manufacturer="Keyboardio"
+keyboardio_imago.build.board=AVR_KEYBOARDIO_IMAGO
+keyboardio_imago.build.core=arduino:arduino
+keyboardio_imago.build.variant=keyboardio-imago
+keyboardio_imago.build.extra_flags={build.usb_flags} '-DKALEIDOSCOPE_HARDWARE_SPEC_H="Kaleidoscope-Hardware-Keyboardio-Imago.h"' 
```

Comment:
Currently, only the Model01 has been given a hardware specification header. In non-virtual builds, all hardwares are still fully functional. Once this PR is accepted, follow up PRs will add virtual build capability to all other hardwares.